cpp/src/parquet/arrow/schema_internal.cc

```diff
@@ -211,14 +212,22 @@ Result<std::shared_ptr<ArrowType>> GetArrowType(Type::type physical_type,
   }
 }
 
+// ARROW-12096 -- Overloading functions with new input (setting default as NANO)
```

Comment:
This comment doesn't seem terribly informative. Can you remove it?

Reply:
Done.